install.sh

```diff
@@ -672,7 +672,18 @@ if [[ -n "${HOMEBREW_ON_LINUX-}" ]]; then
       shell_profile="$HOME/.profile"
       ;;
   esac
+fi
+
+# HOMEBREW_PREFIX is not in PATH on ARM macOS, so remind users to set up their shell
+if [[ -z "${HOMEBREW_ON_LINUX-}" ]] && [[ "$UNAME_MACHINE" == "arm64" ]]; then
+  cat <<EOS
+- Add Homebrew to your ${tty_bold}PATH${tty_reset} in ${tty_underline}${shell_profile}${tty_reset}:
```

Comment:
Maybe worth checking it's not already in the `PATH`?

Reply:
No, I think there's no need. 

Only users who already understand how to use the `PATH` variable are bothered by duplicate `PATH` entries, even if there's no harm to them. Being informed about `brew shellenv` is helpful even for users who know to set their `PATH` too, as it reminds them to set `MANPATH` (and maybe the slightly less useful `INFOPATH`).